Assets/Scripts/AsteroidMovement.cs

```diff
@@ -16,14 +16,8 @@ private void Start()
         speed = Random.Range(minSpeed, maxSpeed);
         appliedInitialForce = false;
         rb = GetComponent<Rigidbody>();
```

Comment:
Can you revert the changes in this file? I already fix this in my ProceduralGeneration branch, so changing it here would just lead to some annoying merge conflicts.

Reply:
Done